packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/TwoStepVerification/GoogleAuth/index.js

```diff
@@ -25,12 +27,15 @@ class GoogleAuthContainer extends React.PureComponent {
     const next = this.props.data.getOrElse({})
     const prev = prevProps.data.getOrElse({})
     if (next.authType !== prev.authType) {
-      this.props.updateUI({ successToggled: true })
-      this.props.triggerSuccess()
-      this.props.goBackOnSuccess()
-      this.props.handleGoBack()
+      this.handleUpdate()
     }
   }
+  handleUpdate () {
```

Comment:
no need for new method if it is only called once. just inline in `componentWillMount`